app/main.py

```diff
@@ -1,2 +1,56 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+
+    def __init__(self, capacity: int = 8, load_factor: float = 2 / 3) -> None:
+        self.capacity = capacity
+        self.load_factor = load_factor
+        self.hash_table: list = [None] * capacity
+
+    def __setitem__(self, key: Any, value: Any) -> None:
+        hash_key = hash(key) % self.capacity
+        while True:
+            if self.hash_table[hash_key] is not None:
+                if self.hash_table[hash_key][0] == key:
+                    self.hash_table[hash_key][1] = value
+                    break
+                else:
+                    hash_key += 1
+            else:
+                self.hash_table[hash_key] = [key, value]
+                break
+
+        if self.__len__() > self.capacity * self.load_factor:
+            self.resize()
+
+    def __getitem__(self, key: Any) -> Any:
+        hash_key = hash(key) % self.capacity
+        while True:
+            if self.hash_table[hash_key] is not None:
+                if self.hash_table[hash_key][0] == key:
+                    return self.hash_table[hash_key][1]
+                else:
+                    hash_key += 1
```

Comment:
Do not forgot to divide for
```suggestion
                    hash_key = (hash_key + 1) % self.capacity
```